rd-net/RdFramework/Tasks/RdCall.cs

```diff
@@ -179,7 +179,7 @@ private IRdTask<TRes> StartInternal(Lifetime requestLifetime, TReq request, [Not
       
       Wire.Send(RdId, (writer) =>
       {
-        SendTrace?.Log($"{task} :: send request: {request.PrintToString()}");
+        SendTrace?.Log($"{task} :: send request: {request?.PrintToString()}");
```

Comment:
_(Same as above)_ `PrintToString` is supposed to handle null better (by printing `"<null>"`), so could we leave this as is?